tests/UI/campaigns/functional/BO/03_catalog/07_discounts/01_cartRules/01_CRUDCartRule.js

```diff
@@ -148,9 +148,9 @@ describe('CRUD cart rule', async () => {
     });
 
     it('should set the promo code', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'setPromoCode_1', baseContext);
+      await testContext.addContextItem(this, 'testIdentifier', 'addPromoCode_1', baseContext);
```

Comment:
I understand but modifying this file seems to be out of scope, since it's in the `functional` campaign ?

Reply:
I just updated the name of the function in the cart rule page, so I have to do it in tests too, to avoid breaking them.
Same thing for tests identifiers to make it more appropriate.